docbot/app.py

```diff
@@ -2,6 +2,8 @@
 from flask import Flask, request
 
 from .handlers import webhook_handler, list_events_handler
+from prometheus_flask_exporter import PrometheusMetrics
```

Comment:
This is a third-party lib, isn't it? It shouldn't be among local imports.

Reply:
Yes, the import has been moved up.